assets/app/features/upgrade/controllers/upgrade-landing.controller.spec.js

```diff
@@ -91,6 +110,33 @@ describe('Upgrade Landing Controller', function() {
                     assert.isFalse(value.status);
                 });
             });
+
+            it('should contain checks for addons', function () {
+                expect(controller.prechecks.checks).toEqual(
+                    jasmine.objectContaining({
+                        'clusters_healthy': jasmine.objectContaining(
+                            { status: jasmine.any(Boolean), label: jasmine.any(String) }
+                        ),
+                        'ceph_healthy': jasmine.objectContaining(
+                            { status: jasmine.any(Boolean), label: jasmine.any(String) }
```

Comment:
If the language keys match checks' keys, we could also assert we're using the right language key for each check.
Take a look at https://github.com/crowbar/crowbar-ui/blob/master/assets/app/features/upgrade/controllers/upgrade-nodes-repositories-checks.controller.spec.js#L306-L346